### DISCUSSION TITLE
Workflow module docs?

### DISCUSSION THREAD

**Discussion by aharonach** [atrocore/atrocore#682] [Q&A]
Hi,
I have the "Workflows" module, but the docs are showing different features from what the module has.
Is there any explanation of how to use the module?

**Comment by atrolex** [comment]
We have 2 "workflows" modules. The first one, for which you see the description, was discontinued, and we created completely a new one. There is still no documentation for the new module. The fist module was oriented on depelopers, the new one is oriented on normal users and administrators.

Current version (at 03.04.23) of the module enables you to listen to create, update, detele in some entity (trigger), check for certain condition, and if the condition is met, execute some action. Following actions are possible now:
1) Execute Import / Export Feed or Connecor
2) Update some field value (i think only for direct relations)

I think the next action to implement will be mass update.

Write to our support it you want to know more.

**Reply by aharonach** [reply]
Thank you!

I must admit that this module's new version is straightforward and I understood my way around. The current docs just got me confused.

One feature in the new module would be great: using another field in the "field value" field, exactly how the "composition" attribute type in the "advanced data types" module is working. And it will be even better to have the option to use a related entity's field value.
For example, I want to create a workflow in which a user creates a child product, the SKU will be automatically updated to the same SKU of the parent with a suffix.

**Reply by atrolex** [reply]
I think this should already be possible, but I don't know how. Please write to our support about it

**Reply by asapko** [reply]
Hi Aharon! Creating a workflow in which a user creates a child product, the SKU will be automatically updated to the same SKU of the parent with a suffix is possible in current module via twig. If you want us to create such twig code, we can do it for 120 euro. It can be payed by custom service in our store or via bill

**Reply by aharonach** [reply]
Using twig is possible in the current version? It's just missing the ability to fetch related record fields?

**Reply by asapko** [reply]
Using twig is possible

**Reply by aharonach** [reply]
Hi asapko,

I tried to use a twig template to fetch a field value for updating some other fields, but it didn't work. For testing, I've set a workflow that updates the product description with its SKU, but I just got the twig template as a literal value, as you could see in my screenshot:

![image](https://user-images.githubusercontent.com/25413975/231838104-a7a3b015-6b95-4186-8816-43d6b5756771.png)

The workflow looks like this:

![image](https://user-images.githubusercontent.com/25413975/231842047-c09c78f1-b2f6-46ce-b303-b40ba99f479b.png)

![image](https://user-images.githubusercontent.com/25413975/231842143-ce0e43a7-dfa9-4776-adff-9194fd9d35f8.png)
